datalad_catalog/webcatalog.py

```diff
@@ -328,6 +393,43 @@ def get_location(self):
         node_fn = node_fn.with_suffix(".json")
         return node_fn
 
+    def get_config(self):
+        """Get Node-level config
+
+        In general a Node instance config can be loaded from file
+        (e.g. "/metadata/dataset_id/dataset_version/config.json")
+
+        If a config file for the Node instance does not exist:
+        - load config content from dataset-level config on the parent
+          catalog instance, and also create the file
+        - if dataset-level config on the parent catalog instance does not exist,
+          inherit/load config from the catalog-level config file.
+        """
+        # Expected config file path
+        dataset_config_path = (
+            self.parent_catalog.metadata_path
+            / self.dataset_id
+            / self.dataset_version
+            / "config.json"
+        )
+        if dataset_config_path.exists() and dataset_config_path.is_file():
```

Comment:
If I understand its [docs](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_file) correctly, `is_file()` should be sufficient
```suggestion
        if dataset_config_path.is_file():
```